graphql-ballerina/listener_utils.bal

```diff
@@ -65,7 +68,10 @@ isolated function processJsonPayload(Engine engine, json payload, http:Response
         operationName = resolveOperationName(operationNameInPayload);
     }
     if (documentString is string) {
-        OutputObject outputObject = engine.getOutputObjectForQuery(documentString, operationName);
+        [OutputObject, string] [outputObject, errorType] = engine.getOutputObjectForQuery(documentString, operationName);
+        if (errorType == "SyntaxError" || errorType == "DocumentError") {
+            response.statusCode = 400;
+        }
```

Comment:
Since setting the bad request status code is duplicated, shall we move it to a separate function and call it?